src/glance.gleam

```diff
@@ -525,6 +525,10 @@ fn module_name(name: String, tokens: Tokens) -> Result(#(String, Tokens), Error)
 fn optional_module_alias(tokens: Tokens) -> #(Option(String), Tokens) {
   case tokens {
     [#(t.As, _), #(t.Name(alias), _), ..tokens] -> #(Some(alias), tokens)
+    [#(t.As, _), #(t.DiscardName(alias), _), ..tokens] -> #(
+      Some("_" <> alias),
```

Comment:
Let's make this an `AssignmentName` to match the other assignment names which can be discards.

Reply:
Fixed, is I properly understanding :)
Changelog updated

Reply:
Perfect!